prow/plugins/cherrypick/cherrypick-unapproved.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package cherrypick adds the `do-not-merge/cherry-pick-not-approved`
+// label to PRs against a release branch which do not have the
+// `cherrypick-approved` label.
+package cherrypick
```

Comment:
Should the name be `cherrypick-unapproved` or something like that?

Reply:
We an [external `cherrypicker` plugin](https://github.com/kubernetes/test-infra/tree/fa801299f10285d4ef491e84bb5f8ce916cf0977/prow/external-plugins/cherrypicker) that implements a `/cherrypick` command so yeah, this package/plugin should be named something else to avoid confusion.

Not sure what off the top of my head though.  Will review and come back